.eslintrc.js

```diff
@@ -257,12 +257,31 @@ module.exports = {
       files: ["frontend/src/**/*"],
       rules: {
         "no-restricted-syntax": [
-        "error",
-        {
-          selector: "Literal[value=/mb-base-color-/]",
-          message: "You may not use base colors in the application, use semantic colors instead. (see colors.module.css)",
-        }
-      ],
+          "error",
+          {
+            selector: "Literal[value=/mb-base-color-/]",
+            message:
+              "You may not use base colors in the application, use semantic colors instead. (see colors.module.css)",
+          },
+        ],
+      },
+    },
+    {
+      files: ["frontend/src/metabase/**/*"],
+      rules: {
+        "no-restricted-syntax": [
+          "error",
+          {
+            selector: "Literal[value=/mb-base-color-/]",
+            message:
+              "You may not use base colors in the application, use semantic colors instead. (see colors.module.css)",
+          },
```

Comment:
Is this rule is duplicated now (it's already set for `frontend/src/**/*`)?

Reply:
yes, it's done like this because overrides literally overrides the pattern and removes previous override